### ISSUE TITLE
Support for ownCloud News app

### ISSUE THREAD

**Issue by jancborchardt** [FredJul/Flym#31]
[Nextcloud](https://nextcloud.com) also has a News reader web app / server with an API: https://github.com/nextcloud/news/tree/master/docs/externalapi cc @Raydiation and @zimba12 are the developers.

Would be cool if FeedEx would support that as well since Nextcloud gets more and more widespread. You can see some screenshots of the app at https://apps.nextcloud.com/apps/news

Cheers! :)

**Comment by jancborchardt** [comment]
Just by the way, meanwhile many of us moved to [Nextcloud](https://nextcloud.com) and we would be happy to help you implement sync support. We are in the chat channel at https://riot.im/app/#/room/#freenode_#nextcloud-dev:matrix.org

**Comment by FredJul** [comment]
FYI, I actually closed all previous tickets to not be spammed with v1 problems and being able to concentrate on v2. However, some tickets like this one should not have been closed, so I reopen it.

Concerning Owncloud I didn't changed my mind: having the support for it would be good. However I'm pretty sure this will be very time consuming and I'm already having difficulties to find time for more basic features. Moreover Flym is initially designed to work without server synchronization, I'm not sure having both behavior will be something easy to do.

In conclusion, if someone want to work on it I'll clearly try to help him. Otherwise I'm afraid I won't focus on that before a while.

**Comment by jakobroehrl** [comment]
Nextcloud Support would be great, I like the App design a lot!

**Comment by jancborchardt** [comment]
As information for anyone who would like to work on Nextcloud integration, we have a Single-Sign-On library for Android which could simply be used: https://github.com/nextcloud/Android-SingleSignOn

**Comment by prog-amateur** [comment]
So any news about Nextcloud SSO in Flym please ? It looks very promising ! Thank you very much.
